docs/site/Examples-and-tutorials.md

```diff
@@ -6,34 +6,37 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Examples-and-tutorials.html
 ---
 
-LoopBack 4 comes with the following example projects:
-
-- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
-  Tutorial on setting up a simple hello-world application using LoopBack 4.
+LoopBack 4 comes with the following tutorials:
 
 - **[todo](todo-tutorial.md)**: Tutorial on building a simple application with
   LoopBack 4 key concepts using bottom-up approach.
 
 - **[todo-list](todo-list-tutorial.md)**: Tutorial on introducing related models
   and building their API from the Todo tutorial
 
+- **[soap-calculator](soap-calculator-tutorial.md)**: Tutorial on integrating
+  SOAP web services.
+- **[hello-world](https://github.com/strongloop/loopback-next/tree/master/examples/hello-world)**:
```

Comment:
We are listing all individual tutorials, but then pointing to another page for a list of examples. I would prefer to be more consistent and do the same for tutorials and examples.

In my experience, when the same information is maintained in multiple places (Examples-and-tutorials, Examples, Tutorials), these places eventually get out of sync because people will often make change in one place only and forget about the other places.

I am proposing to trim down Examples-and-tutorials and maintain the list of tutorial inside `Tutorials.md` only.

Conceptually:

```md
LoopBack 4 comes with tutorials, which are available here:
- **[Tutorials](Tutorials.md)**

LoopBack 4 also comes with example projects, which are available here:
- **[Example Projects](Examples.md)**
```

Having said that, this deep structure may be less friendly to our readers, they will have to make one more click to find the example/tutorial they are looking for. Two alternatives to consider:

- Don't create additional level of nesting. Keep both tutorials & examples in the same sidebar. Use sections in `Examples-and-tutorials` to split the list in multiple parts - tutorials, examples, contributions.

- Leverage `{% include %}` to keep the list of examples/tutorials in a single file, and render this list on multiple pages (Examples-and-tutorials, Examples, Tutorials). Prior art in LB 3.x docs: [Remote Methods](https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/lb3/Remote-methods.md#options) and [rm-options.md](https://github.com/strongloop/loopback.io/blob/069bbf7cdc7660c0a7cc57a9d9ebab09aeed4e39/_includes/content/rm-options.md). Personally, I find this setup a bit confusing, because it takes me time to figure out where to locate the included file.

Let's discuss!

/cc @dhmlau @strongloop/sq-lb-apex

Reply:
You need to decide how important it is to separate Examples and Tutorials. I like the idea of having one page to link to the 3 sections, rather than separate landing pages.

Reply:
- I don't really mind, at first, whether it is `Examples and tutorials` and having a nested `Examples` and `Tutorials`.  After a second thought, if we'll be showing the step as the nested item under tutorials, it might be good that we have `Examples` and `Tutorials` as the top level element. 

- For tutorials (if those really fit, see [my comment below](https://github.com/strongloop/loopback-next/pull/2327#discussion_r255175476)), i think it makes sense to be consistent to have a similar format as the todo tutorial.

Reply:
I would like to see:
- examples and tutorials list on separate pages
- tutorials are kept in docs and example steps are kept in each example repository

The menu structure would be:

- Tutorials
  - Todo: a page with todo tutorials
  - TodoList: a page with todo list tutorials
  - ...(more tutorials)
- Examples: We only need one page to list all available examples **without** sub menus. And the example repository keeps all the details in its `.md` file

Reply:
HI @jannyHou the description in your comment is what the current design is for this PR (except for the Contibutions section which you don't mention, but what you described for Tuts and Ex is what is contained in current PR)

* examples and tutorials list on separate pages
* tutorials are kept in docs and example steps are kept in each example repository

The menu structure would be:

* Tutorials
  
  * Todo: a page with todo tutorials
  * TodoList: a page with todo list tutorials
  * ...(more tutorials)
* Examples: We only need one page to list all available examples **without** sub menus. And the example repository keeps all the details in its `.md` file